BraveShared/BraveStrings.swift

```diff
@@ -886,6 +886,7 @@ extension Strings {
     public static let cache = NSLocalizedString("Cache", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Cache", comment: "Settings item for clearing the cache")
     public static let cookies = NSLocalizedString("Cookies", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Cookies and Site Data", comment: "Settings item for clearing cookies and site data")
     public static let findInPage = NSLocalizedString("FindInPage", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Find in Page", comment: "Share action title")
+    public static let searchWithBrave = NSLocalizedString("SearchWithBrave", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Search with Brave", comment: "Share action title")
```

Comment:
Make the `comment` more descriptive, it helps our translators to use correct words in other languages.
Example of a good comment arg: `Title of an action that allows user to select a text and perform a web search query for it`